mysqld_exporter.go

```diff
@@ -571,6 +624,93 @@ func (e *Exporter) scrape(ch chan<- prometheus.Metric) {
 			}
 		}
 	}
+
+	if *perfEventsStatements {
+		// Timers here are returned in picoseconds.
+		perfSchemaEventsStatementsRows, err := db.Query(`
+SELECT
```

Comment:
I'd like to fix that in a separate PR, I plan to cleanup all those queries into the const block.